spec/import_spec.rb

```diff
@@ -123,10 +170,10 @@ def upload_file!(name, ext='csv')
     context "having authors with the same Id" do
       before do
         add_author_resource({
-          before_batch_import: ->(importer) do
-            Author.where(id: importer.values_at("id")).delete_all
-          end
-        })
+                                before_batch_import: ->(importer) do
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.